<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>279</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>279</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设备属性</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>220</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>220</height>
        </size>
       </property>
       <property name="title">
        <string>设备属性</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>设备名称</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QLineEdit" name="channelNumber">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>用 户 名</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>序 列 号</string>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QLineEdit" name="port"/>
          </item>
          <item row="2" column="6">
           <widget class="QLineEdit" name="password">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>端口号</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>通道树</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="3">
           <widget class="QLineEdit" name="deviceName"/>
          </item>
          <item row="1" column="2" colspan="3">
           <widget class="QLineEdit" name="ipAddr"/>
          </item>
          <item row="2" column="5">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>密  码</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" colspan="3">
           <widget class="QLineEdit" name="userName"/>
          </item>
          <item row="3" column="2" colspan="5">
           <widget class="QLineEdit" name="serialNumber">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>IP 地 址</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>131</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnOk">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
